README.md

```diff
@@ -267,6 +267,26 @@ const element = <Component />
 areComponentsEqual(element.type, Component) // true
 ```
 
+Another way - compare "rendered" element type
+
+```js
+const element = <Component />
+console.log(element.type === (<Component/>).type) // true
+
+// better - precache rendered type
+const element = <Component />
```

Comment:
Incompatible with PropTypes

Reply:
:( Yeah, now I recall all the issues around it.